README.md

```diff
@@ -32,53 +45,102 @@ This software should :
 
 *Tick the tasks once successfully implemented and tested*
 
-It is assumed that the sensors' data is not processed in any way by the embedded computer. The data received is raw and should be converted to readable values by the ground station.
-
->NB : how do we deal with the calibration of the sensors ?
-
 
 # Hardware
 
-The ground station is made of different systems :
+The hardware is made of the following elements :
+  * Rocket

```

Comment:
Shouldn't it be "Rocket TM", if at all? I mean, the GS is just receiving from the rocket and sends command through the LPS, but this still "outside" the GS in my opinion.